e2e/stack.go

```diff
@@ -91,7 +90,14 @@ func (s *Stack) Run(ctx context.Context, env *environment.Env) (*StackConfig, er
 		return nil, fmt.Errorf("new deploy config: %v", err)
 	}
 	deployConfig.SetDeployments(l1Deployments)
+	deployConfig.L1GenesisBlockTimestamp = hexutil.Uint64(time.Now().Unix())
 	deployConfig.L1UseClique = false // Allows node to produce blocks without addition config. Clique is a PoA config.
+	deployConfig.SequencerWindowSize = 4
```

Comment:
could you add a brief comment here for transparency describing why the sequencer window size needs to be set as 4?